docs/content/what-is-a-schedule-to-start-timeout.md

```diff
@@ -10,24 +10,37 @@ tags:
 import CenteredImage from "../components/CenteredImage.js"
 import RelatedReadList from '../components/RelatedReadList.js'
 
-A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts executing that Activity Task.
+A Schedule-To-Start Timeout is the maximum amount of time that is allowed from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (that is, placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts (that is, picks up from the Task Queue) that Activity Task.
+In other words, it's a limit for how long an Activity Task can be enqueued.
+
+The moment that the Task is picked by the Worker from the Task Queue is considered to be the start of the Activity Task for the purposes of the Schedule-To-Start Timeout and associated metrics.
+This definition of "Start" avoids issues that a clock difference between the Temporal Cluster and a Worker might create.
 
 <CenteredImage
 imagePath="/diagrams/schedule-to-start-timeout.svg"
 imageSize="100"
 title="Schedule-To-Start Timeout period"
 />
 
-A [Retry Policy](/docs/content/what-is-a-retry-policy) attached to an Activity Execution retries an Activity Task Execution.
-Thus the Schedule-To-Start Timeout is applied to each Activity Task Execution within an Activity Execution.
 
+:::warning "Schedule" in Schedule-To-Start and Schedule-To-Close may reference different moments
+
+Schedule-To-Start Timeout is enforced for each Activity Task, whereas Schedule-To-Close Timeout is enforced once per Activity Execution.
+Thus, "Schedule" in Schedule-To-Start refers to the scheduling moment of *every* Activity Task in the sequence of Activity Tasks that make up the Activity Execution, while
+"Schedule" in Schedule-To-Close refers to the *first* Activity Task in that sequence.
+
+:::
```

Comment:
```suggestion
"Schedule" in Schedule-To-Start and Schedule-To-Close have different frequency guarantees.

The Schedule-To-Start Timeout is enforced for each Activity Task, whereas the Schedule-To-Close Timeout is enforced once per Activity Execution.
Thus, "Schedule" in Schedule-To-Start refers to the scheduling moment of *every* Activity Task in the sequence of Activity Tasks that make up the Activity Execution, while
"Schedule" in Schedule-To-Close refers to the *first* Activity Task in that sequence.
```

Like I said before, let's stay away from the admonitions - this is perfectly suitable core information.